packages/stark-ui/src/modules/table/components/column.component.ts

```diff
@@ -130,6 +130,27 @@ export class StarkTableColumnComponent extends AbstractStarkUiComponent implemen
 	 */
 	private _headerLabel?: string;
 
+	/**
+	 * Value to be shown as the column's footer.
+	 */
+	@Input()
+	public set footerValue(value: any) {
```

Comment:
Same here: use `string|number` instead of `any` ;)

Reply:
Yeah at first It was what I used, but I was afraid to be to strict. Maybe a day we'll need an object. But I agree, for now I'll put it as string|number

Reply:
Indeed... cause maybe later on we'll need to implement a formatter or something to display more complex values...